### ISSUE TITLE
Suggestion: Member requirement

### ISSUE THREAD

**Issue by VerumUK** [MarkehMe/FactionsWarps#9]
An idea for a future update, allow us to have an option in config to set a member requirement for Factions to get access to the faction warps, this is to avoid a new person joining, making a 1 man faction and simply using it for warps.

**Comment by markhughes** [comment]
Great idea, I can see how this would be useful.

---

 Here is the proposed configuration option, plus a few extras:

_(everything by default will be set to 0)_

``` javascript
"reqMinimumMembers": 0, // minimum members required to use create/use warps
"reqMinimumFactionPower": 0.0, // minimum faction power required 
"reqMinimumClaims": 0, // minimum chunks claimed required 
```

**Update 25th May** - And some configuration options:

``` javascript
"reqBlocksSetWarp": true, 
"reqBlocksUseWarp": true,
```

**Comment by VerumUK** [comment]
Ah yeah that looks good, I like the idea of minimum claims required. Perhaps have it so if they fall below the threshold it makes set warps unusable. If you can :)

**Comment by markhughes** [comment]
Sure. I updated my post with these configuration option ideas:

``` javascript
"reqBlocksSetWarp": true, 
"reqBlocksUseWarp": true,
```

**Comment by markhughes** [comment]
Implemented in a772e5c029f64a5b561dfbb1e071580e81479b65
